libnavigation-base/src/main/java/com/mapbox/navigation/base/trip/model/alert/CountryBorderCrossingAdminInfo.kt

```diff
@@ -3,21 +3,21 @@ package com.mapbox.navigation.base.trip.model.alert
 /**
  * Administrative information.
  */
-class BorderCrossingAdminInfo private constructor(
+class CountryBorderCrossingAdminInfo private constructor(
```

Comment:
Would it be better to have one struct BorderCrossingAdminInfo having codes for countries in case we cross country or codes for states in case we detect state boundary crossing?
In NN we can have just RouteAlertAdminInfo in that case.